code/target/src/main/java/com/adobe/marketing/mobile/target/TargetConstants.java

```diff
@@ -268,6 +268,104 @@ static class PreviewKeys {
         static final String DEEPLINK_SCHEME = "adbinapp";
         static final String DEEPLINK_SCHEME_PATH_CANCEL = "cancel";
         static final String DEEPLINK_SCHEME_PATH_CONFIRM = "confirm";
+
+        /** Base64 encoded button PNG */
+        static final String ENCODED_BUTTON_BACKGROUND_PNG =
```

Comment:
Recommend creating a text file with this string in the `assets` folder of the extension and then using 
`StreamUtils.readAsString(context.assets.open("your_icon.txt"))` for a simpler contestants file.